### ISSUE TITLE
Non-resizable IntelliJ IDEA debugging tooltips

### ISSUE THREAD

**Issue by tsdh** [swaywm/sway#5959]
Im running Sway and wlroots from their current git master branches.  The merge of the PRs #5948 and swaywm/wlroots#2605 by @BrassyPanache finally made IntelliJ usable with sway for me.  There's just the following tiny bit which I still run into.

When debugging a Java program and standing on a breakpoint, you can hover over a variable and a tooltip will appear with a big + symbol.  Clicking that will open a new inspection tooltip where you can expand and view the structure of the object.  This tooltip is resizable in Gnome (on Wayland) but it's not resizable in Sway.  Or rather, I could initially make it smaller by clicking and holding an edge and dragging but now that it is pretty small (see screenshot), I cannot make it bigger (or even smaller) again.

I've tried grabbing on the edges (where the mouse pointer gets the resize-shape) and pressing $mod while dragging with the right mouse button.  The former just does nothing.  With the latter, it seems pressing and holding the right mouse button delivers a right click event to the tooltip, at least I get the same IntelliJ context menu I get when just right-clicking in there.

![intellij_debug_popup](https://user-images.githubusercontent.com/103854/105182785-bf88c680-5b2d-11eb-8ad7-174cc658c2e2.png)

**Comment by cpouldev** [comment]
Did you ever find a solution for this issue? Using the latest sway-git and jetbrains products I'm facing the same issue but for the "Search Everywhere" popup

**Comment by tsdh** [comment]
@cpouldev No, sadly not.  I can resize the "Search Everywhere" popup one or two millimeters using $mod and right-click dragging at its borders but then it stops resizing.

**Comment by cpouldev** [comment]
Meh.. 

I opened an issue on YouTrack (https://youtrack.jetbrains.com/issue/IDEA-316500), with a screencast showing the resizing issue but I don't think they will ever deal with it.

Their response:

> sadly, we have limited capabilities of supporting different window managers. We expect Idea to be run on KDE, Gnome, or Unity. Generally, we try to support as many window managers as possible, but in this particular case, I was not able to install Sway, and in other tiling managers, such as i3, the issue is not reproduced.

**Comment by tsdh** [comment]
Yeah, the behavior for me is exactly as shown in your screencast.  I'm a very lucky guy as my "search everywhere" popup is much larger as yours so I have no pressing need to resize it. :-)

**Comment by tsdh** [comment]
Oh no, now my "search everything" popup shrunk to some too small size, so now I'm actually also very negatively affected by this issue. :-(

I'm using sway 1.9 right now.

**Comment by vanarok** [comment]
I'm also facing this problem

**Comment by japalvia** [comment]
I've had this issue for a long time in CLion as well. It occurs inconsistently but once the search dialog becomes small it usually persists that way.

**Comment by xbasel** [comment]
Starting CLion/IntelliJ with Wayland (`-Dawt.toolkit.name=WLToolkit`) allows resizing, but some popups always open shrunk and need manual resizing.

**Comment by tsdh** [comment]
Yup, `-Dawt.toolkit.name=WLToolkit` is not too bad.  I feel the fonts are sharper.  But indeed, some popup positioning is quite off.  And I get a crash when the window title changes to something with an Umlaut (äöü) because those seem to be passed as something which is not valid UTF-8 to wayland.  (As a workaround, I renamed my data sources with umlauts in their name.)

**Comment by false-vacuum** [comment]
-Dawt.toolkit.name=WLToolkit really sucks still with jetbrains. Has anyone else found another workaround for this?
